src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/endpoints/CaseDocumentAmController.java

```diff
@@ -1,211 +1,182 @@
+import static uk.gov.hmcts.reform.ccd.document.am.apihelper.Constants.INPUT_STRING_PATTERN;
+
+@Api(value = "cases")
+@RestController
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
-public class CaseDocumentAmController implements CaseDocumentAm {
+public class CaseDocumentAmController  {
 
     private static final Logger LOG = LoggerFactory.getLogger(CaseDocumentAmController.class);
 
-    private DocumentManagementService  documentManagementService;
+    private transient DocumentManagementService  documentManagementService;
 
     @Autowired
     public CaseDocumentAmController(DocumentManagementService documentManagementService) {
         this.documentManagementService = documentManagementService;
     }
 
-    @Override
+    //******************** Delete API ************
+    @ApiOperation(value = "Deletes a Case Document.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 204, message = "No Content")
+       })
+    @RequestMapping(value = "/cases/documents/{documentId}",
+        produces = {APPLICATION_JSON},
+        method = RequestMethod.DELETE)
     public ResponseEntity<Object> deleteDocumentbyDocumentId(
-        @ApiParam(value = "Service Auth (S2S). Use it when accessing the API on App Tier level.", required = true)
-        @RequestHeader(value = "ServiceAuthorization", required = true) String serviceAuthorization,
-
-        @ApiParam("Authorization header of the currently authenticated user")
-        @RequestHeader(value = "Authorization", required = true) String authorization,
-
-        @ApiParam("documentId")
         @PathVariable("documentId") UUID documentId,
-
-        @ApiParam("user-id of the currently authenticated user. If provided will be used to populate the creator field of a document"
-            + " and will be used for authorisation.")
         @RequestHeader(value = "user-id", required = false) String userId,
-
-        @ApiParam("Comma-separated list of roles of the currently authenticated user. If provided will be used for authorisation.")
         @RequestHeader(value = "user-roles", required = false) String userRoles,
-
-        @ApiParam("permanent delete flag")
         @Valid @RequestParam(value = "permanent", required = false, defaultValue = "false") Boolean permanent) {
 
         ResponseEntity responseEntity = documentManagementService.getDocumentMetadata(documentId);
-        if (documentManagementService.checkUserPermission(responseEntity, documentId, authorization, Permission.UPDATE)) {
+        if (documentManagementService.checkUserPermission(responseEntity, documentId, Permission.UPDATE)) {
             return  documentManagementService.deleteDocument(documentId, userId, userRoles, permanent);
 
         }
-        LOG.error("User doesn't have update permission on requested document {}",  HttpStatus.FORBIDDEN);
+        LOG.error("User doesn't have update permission on requested document " + HttpStatus.FORBIDDEN);
```

Comment:
These + symbols were removed as part of sonar code smell issue, please revert this.

Reply:
Done.